torch/_dynamo/convert_frame.py

```diff
@@ -473,7 +473,9 @@ def _compile(
     output: Optional[OutputGraph] = None
     # This is shared across restarts
     mutated_closure_cell_contents: Set[str] = set()
+    fail_type: Optional[str] = None
     fail_reason: Optional[str] = None
+    fail_user_frame_summary: Optional[str] = None
```

Comment:
Pls check the output example at this PR's description. ```fail_user_frame_summary``` is the innermost user frame summary from the exception. It includes filename and lineno. I can further split this into two column ```fail_user_frame_filename``` and ```fail_user_frame_lineno``` if it's needed.

Reply:
Split seems much better

Reply:
Done